PoseEstimation-TFLiteSwift/StillImageLineViewController.swift

```diff
@@ -139,16 +198,37 @@ class StillImageLineViewController: UIViewController {
         navigationController?.present(pickerVC, animated: true)
     }
     
-    @IBAction func changeThreshold(_ sender: UISlider) {
-        threshold = (sender.value == sender.minimumValue) ? nil : sender.value
-        if let threshold = threshold {
-            thresholdLabel?.text = String(format: "%.2f", threshold)
-        } else {
-            thresholdLabel?.text = "nil"
-        }
-        
-        let partIndex = selectedPartIndex
-        guard let output = poseEstimator.postprocessOnLastOutput(with: threshold, on: partIndex) else { return }
+    @IBAction func didChangeHumanType(_ sender: UISegmentedControl) {
+        isSinglePerson = (sender.selectedSegmentIndex == 0)
+        updateOverlayViewWithOnlyPostprocess()
+    }
+    
+    @IBAction func didChangeDimension(_ sender: UISegmentedControl) {
+        // NOT SUPPORT YET
```

Comment:
if It will be implemented, It is better using `todo` comment. It is easy to track the unimplemented codes.

Reply:
@MBKwon

Did you mean [`// TODO`](https://stackoverflow.com/a/16913148/4160632) annotation? I agree. Thank you for your feedback.